### ISSUE TITLE
Evaluate possibility of removing notifications about report expiration

### ISSUE THREAD

**Issue by evilaliv3** [globaleaks/GlobaLeaks#3322]
This ticket is to evaluate the possibility of removing notifications about reports expiration and evaluate alternatives to communicate this information more efficiently.

The consideration at the base of this proposal is that an active system, running for many years, will have a large set of reports expiring every day and thus triggering notifications about expiration.

On such a situation, users would be continuously notified via email with the risk of being annoyed and start ignoring email notifications.

**Comment by evilaliv3** [comment]
I consider that based on the fact that the notifications about new, updated and unread updates are working quite well, we could probably remove the functionality of notification about expiring reports could be fully deprecated.

@giorgiofraschini @elbill @danielvaknine @larrykind @maxmois @: what do you think?

**Comment by danielvaknine** [comment]
I don't think reminder for expiration date should be removed yet, at least
not until a full-feathered reminder-function is live. Maybe I'm
misunderstanding though.

**Comment by evilaliv3** [comment]
Thank you for your feedback @danielvaknine

I agree for the moment to keep it, and plan for removing it.

The scenario i would like to discuss and address with you: suppose as an example that the system reaches 10000 reports, with 1000 due to expiration. On such a condition with the current notification mechanism it is likely that the system will notify the same user about the expiration of reports every day. Think how annoyed will be a user just after one week receiving an email saying: "There are 999 reports expiring and the earliest expiration date is tomorrow"

What do you think?

**Comment by danielvaknine** [comment]
Yes, I understand your point!

I think that case would be relatively rare, and instead of removing the
notification all together, I think it would be better to give the
administrator the possibility to disable either the expiration date itself
or just disable the notification to the recipient.

Generally speaking, I think the email notifications are important to help
recipients handle reports in a correct way, which is ultimately also in the
interest of the whistleblower(s).


Removing such notifications could, while being better for a small number of
organisations with ridiculously many reports, instead heart the majority of
organisations in my opinion.

Giving the option to disable (both expiration date and the notification) is
probably the best way to keep all users satisfied!

**Comment by giorgiofraschini** [comment]
I agree with @danielvaknine. Your case scenario @evilaliv3 is extremely rare and it would be enough if the user could just choose to deactivate notifications. I think another development could be leaving the admin to choose when to receive the notification (7 days?, 1 day? everyday since 7 days?)

**Comment by elbill** [comment]
Most users do not feel safe with expiring reports. They would feel even less safe without notifications. We should keep them at the moment.
